extension/src/background/messageListener/__tests__/popupMessageListener.test.js

```diff
@@ -1,6 +1,7 @@
 import { SERVICE_TYPES } from "@shared/constants/services";
 import { popupMessageListener } from "background/messageListener/popupMessageListener";
 import { store } from "background/store";
+import { sessionSlice } from "background/ducks/session";
```

Comment:
I added some test cases here to be doubly sure that background functions are working as expected